### ISSUE TITLE
Face Detect Recognition Verification

### ISSUE THREAD

**Issue by yushenxiang** [yushenxiang/MachineLearningTips#16]

**Comment by yushenxiang** [comment]
https://blog.csdn.net/mogebuyi/article/details/77692862
人脸识别：Deep Face Recognition论文阅读

**Comment by yushenxiang** [comment]
https://zhuanlan.zhihu.com/p/24837264
谷歌人脸识别系统FaceNet解析


https://blog.csdn.net/stdcoutzyx/article/details/46687471
FaceNet--Google的人脸识别

https://blog.csdn.net/qq_15192373/article/details/78490726
Facenet

https://www.cnblogs.com/dyufei/p/8250484.html
TensorFlow环境 人脸识别 FaceNet 应用（一）验证测试集

	
FaceNet是用
https://blog.csdn.net/a13662080711/article/details/80335333

http://www.bubuko.com/infodetail-2401364.html
facenet 代码阅读笔记：如何训练基于triplet-loss的模型

**Comment by yushenxiang** [comment]
https://blog.csdn.net/yuanchheneducn/article/details/51034463
DeepID1 DeepID2 DeepID2+ DeepID3

**Comment by yushenxiang** [comment]
https://blog.csdn.net/lff1208/article/details/77328357
MTCNN-将多任务级联卷积神经网络用于人脸检测和对齐

https://blog.csdn.net/qq_14845119/article/details/52680940
**MTCNN（Multi-task convolutional neural networks）人脸对齐**

https://blog.csdn.net/qq_30159015/article/details/79699855
MTCNN笔记

https://blog.csdn.net/u014380165/article/details/78906898
MTCNN算法及代码笔记

https://blog.csdn.net/sinat_24143931/article/details/78778656
基于NCNN的人脸检测MTCNN实现过程

**Comment by yushenxiang** [comment]
https://zhuanlan.zhihu.com/p/25025596
基于mtcnn和facenet的实时人脸检测与识别系统开发

**Comment by yushenxiang** [comment]
https://blog.csdn.net/u014380165/article/details/76946339
损失函数改进之Center Loss

**Comment by yushenxiang** [comment]
https://blog.csdn.net/u014230646/article/details/79487720
人脸识别（四）：2018 ArcFace: Additive Angular Margin Loss for Deep Face Recognition

**Comment by yushenxiang** [comment]
https://blog.csdn.net/u014365862/article/details/79948612
CosFace中的cos loss（Large Margin Cosine Loss）实现（tensorflow）

**Comment by yushenxiang** [comment]
https://blog.csdn.net/u014380165/article/details/80645489
ArcFace算法笔记

**Comment by yushenxiang** [comment]
https://blog.csdn.net/u014380165/article/details/76946339
损失函数改进之Center Loss
https://blog.csdn.net/u014230646/article/details/53764079
人脸识别（一）A Discriminative Feature Learning Approach for Deep Face Recognition
https://blog.csdn.net/u014380165/article/details/71181256
softmax，softmax-loss，BP的解释

https://blog.csdn.net/u014380165/article/details/76864572
损失函数改进之Large-Margin Softmax Loss

https://blog.csdn.net/shaoxiaohu1/article/details/53325945
论文笔记】Large-Margin Softmax Loss for Convolutional Neural Networks

损失函数改进方法之A-softmax loss
https://blog.csdn.net/sinat_24143931/article/details/79033521

https://www.cnblogs.com/heguanyou/p/7503025.html
A-Softmax的总结及与L-Softmax的对比——SphereFace

https://blog.csdn.net/u014365862/article/details/79948612
CosFace中的cos loss（Large Margin Cosine Loss）实现（tensorflow）


https://blog.csdn.net/u014380165/article/details/80645489
ArcFace算法笔记

https://blog.csdn.net/fuwenyan/article/details/79657738
【论文理解】ArcFace: Additive Angular Margin Loss for Deep Face Recognition（InsightFace）


人脸识别-损失函数改进方法之L-softmax loss center-loss A-softmax loss focal loss
https://blog.csdn.net/u011808673/article/details/80491361

https://blog.csdn.net/fire_light_/article/details/79602310
人脸识别系列（十六）：AMSoftmax

https://blog.csdn.net/shaoxiaohu1/article/details/79139039
【论文笔记】Additive Margin Softmax for Face Verification

https://blog.csdn.net/yjy728/article/details/79730716
AM-Softmax的keras实现: 《Additive Margin Softmax for Face Verification》
